docs/docs/usage/scaladoc/settings.md

```diff
@@ -134,3 +142,38 @@ Example JSON file:
 }
 ```
 
+##### -snippet-compiler
+
+Snippet compiler arguments provide a way to configure snippet checking.
```

Comment:
`checking` sounds a vague term. Compilation is maybe more precise.

Reply:
“type checking” would also work, imho.